pkg/database/metrics.go

```diff
@@ -0,0 +1,33 @@
+package database
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/recover"
+)
+
+func (db Database) emitMetrics(ctx context.Context) {
+	t := time.NewTicker(time.Minute)
+	defer t.Stop()
+
+	for {
+		i, err := db.OpenShiftClusters.QueueLength(ctx)
+		if err != nil {
+			db.log.Error(err)
+		}
+		defer recover.Panic(db.log)
```

Comment:
should be first line in goroutine - the purpose is to prevent an arbitrary panic in a goroutine from bringing the whole process down